src/universal/modules/meeting/components/AgendaCard/AgendaCard.js

```diff
@@ -152,6 +151,7 @@ export default class AgendaCard extends Component {
                 isProject={isProject}
                 name={outcomeId}
                 outcome={outcome}
+                doFocus={!content}
```

Comment:
when you blur an empty card, it goes away. this makes it _really_ difficult to accidentally create 2 outcomes, but multiple people can create some at the same time.